bootstrap/mocks.go

```diff
@@ -0,0 +1,86 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ./interface.go
+
+// Package bootstrap is a generated GoMock package.
```

Comment:
Probably we don't want this package comment, you can avoid writing it by using `-write_package_comment false` when generating mocks

Reply:
thanks for the tip. done